package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "eslint-config-fluid",
+    "version": "0.1.0",
+    "description": "fluid-project eslint configuration",
+    "main": "index.js",
+    "engines": {
+        "node": ">=4.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/fluid-project/eslint-config-fluid.git"
+    },
+    "keywords": [
+        "eslint",
+        "eslintconfig",
+        "fluid",
+        "fluid-project"
+    ],
+    "author": "Fluid Project",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.fluidproject.org/projects/FLUID/issues"
+    },
+    "homepage": "http://fluidproject.org",
+    "peerDependencies": {
+        "fluid-eslint": ">=2.0.0"
```

Comment:
I think we can remove the validation warning since I don't think it is doing much for us and is probably actively misleading. The npm team have acknowledged that peerDependencies is a broken system and have deprecated it in npm 3. The reason for our fork is that the processing of .eslintignore is broken in the main line edition.